portfolio/src/main/webapp/index.html

```diff
@@ -3,16 +3,78 @@
   <head>
     <meta charset="UTF-8">
     <title>My Portfolio</title>
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/boot
+    strap/4.0.0/css/bootstrap.min.css" integrity="sha384-Gn5384xqQ1ao
+    WXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm" crossorigin
+    ="anonymous">
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
   </head>
   <body>
-    <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
-    </div>
+    <section id="home" class="page">
+      <h1>Toba Olokungbemi</h1>
+    </section>
+    <section id="about" class="page">
+      <h2>Yeo! Welcome to my page!</h2>
+      <p>My name is Toba olokungbemi. I am rising Junior at
+      Harvard studying Science with a Secondary in African
+      Studies. I enjoy listening to chill music, afrobeat
+      dancing, traveling, and creating projects. Take a look
+      through the site to learn more!</p>
+      <p>Click the button to get one of my favortie movie/tv quotes!<br>
+        <button class="btn btn-primary" onclick="addRandomQuote()">Quote</button>
+        <div id="quote-container"></div>
+      </p>
+    </section>
+    <section id="life" class="extended-page">
+      <h1>Life Before Quarantine</h1>
+      <div class="row">
```

Comment:
Nit: I'm having a bit of trouble with the "row" naming, since it would suggest that there will be multiple rows. I'm seeing it more as a container for the whole gallery, so I'd probably go with something more like "gallery"...or similar. :)